src/main/java/org/infoshareacademy/javaholics/Route.java

```diff
@@ -0,0 +1,113 @@
+package org.infoshareacademy.javaholics;
+
+import java.io.File;
+
+public class Route {
+
+    private long id;
+    private String name;
+    private String locality;
+    private String placeStart;
+    private String placeStop;
+    private RouteDifficulty difficulty;
+    private File routeFile;
+    private Event lastEvent;
```

Comment:
Musze dodać do siebie

Reply:
pytanie czy nie powinna być to lokalna zmianna wyswietlajac ostani event dla tej trasy, bo w samej klasie lastEvent na co ma wskazywać :)